statsproxy.go

```diff
@@ -47,134 +33,50 @@ func monitor() {
 		defer pprof.StopCPUProfile()
 	}
 
-	ticker := time.NewTicker(time.Duration(config.Service.TickerPeriod) * time.Second)
 signalLoop:
```

Comment:
Do you want this loop to block?  If not, you might want to add a `default:` case to your switch statement below so that it bypasses it if there is nothing on the channels.